changelog/0015-invoke-celery-directly.yml

```diff
@@ -0,0 +1,17 @@
+title: Update supervisor confs to invoke celery directly
+key: invoke-celery-directly
+date: 2019-02-22
+optional_per_env: no
+min_commcare_version: f67f9c8149bd71712f457c270b03b5a55077c273
+max_commcare_version:
+context: |
+  Upgrading to celery 4.x requires removing the dependency on
+  django-celery, which means that the celery management command
+  becomes unavailable.  This prepares for that by invoking the
+  celery command directly.
+
```

Comment:
Nit: adding a `detail:` field here would get rid of the bare "None" in the markdown output below. I think this would be a good place to go over some some details of the importance of the rollout sequence. Something that will call out the importance of sequencing things right and what goes wrong if you don't.

Reply:
@dannyroberts updated